source/includes/api/tables/progress-response.rst

```diff
@@ -78,6 +87,15 @@
        and divide the result by the value of the ``estimatedTotalBytes`` field
        . Then, multiply the result by 100.
 
+       .. note::
+
+         ``mongosync`` approximates the estimated number of copied bytes
+         prior to migration and does not update this value during the
+         synchronization process. This value does not reflect changes
+         made to the source cluster during sync and is not an accurate
+         indicator of migration progress. 
```

Comment:
This should say something like:

mongosync only reflects the bytes copied by the current instance of mongosync. The value does not reflect bytes copied by other instances of mongosync and can become larger than estimatedTotalBytes due to retried operations. This value is not an accurate indicator of migration progress.

Reply:
Ahh apologies, I didn't see that the top part of the description for `estimatedCopiedBytes`. Then there is no need to restate "This value only reflects the bytes copied by the current instance of mongosync".

But the following section is incorrect and should be removed. mongosync does in fact compute the estimated number of copied bytes during the migration by adding up the size of documents that it copies. This value will reflect changes made to the source cluster during sync and can be higher than ``estimatedTotalBytes`` if documents are added to the source or lower if documents are deleted from the source.
```
         mongosync approximates the estimated number of copied bytes
         prior to migration and does not update this value during the
         synchronization process. This value does not reflect changes
         made to the source cluster during sync and is not an accurate
         indicator of migration progress.
```

Reply:
no worries, just made your suggested change!